governance-exercises/Adjust Spork D3M Parameters and DAI Interest Rate -- July 24, 2023.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Adjust Spork Protocol D3M Parameters and DAI Interest Rate - July 24, 2023
+summary: Express your support or opposition to adjusting the Spork DAI Direct Deposit Module (D3M) parameters and DAI Interest Rate.
+discussion_link: https://forum.makerdao.com/t/xxx
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-07-24 16:00 UTC
+end_date: 2023-07-27 16:00 UTC
+---
+# Poll: Adjust Spork D3M Parameters and DAI Interest Rate - July 24, 2023
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling). The community can vote in this poll to express support or opposition to updating the Spork Direct Deposit DAI Module (D3M) with detailed parameters. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, July 24 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to making the following parameter changes to the Spork DAI Direct Deposit Module (D3M) and Interest Rate:
+
+  ### Adjust Spork Protocol D3M Parameters and DAI Interest Rate - July 24, 2023:
+  - Increase Spork D3M Debt Ceiling by 50 million from 50 million to 100 million DAI.
+  - Increase Spork D3M Target Available Debt (gap) by 10 million from 5 million to 15 million.
+  - Adjust Spork Protocol DAI Interest Rate to increase the DAI deposit rate by 0.5% from 2% to 2.5%
```

Comment:
Also a little weird to have this one in the middle. The other three are much more closely related.